app/assets/stylesheets/footer.scss

```diff
@@ -0,0 +1,7 @@
+footer {
+  text-align: center;
```

Comment:
Properties should be ordered font-size, font-weight, padding-bottom, text-align, text-transform